Sources/armory/logicnode/ColorgradingGetShadowNode.hx

```diff
@@ -6,24 +6,15 @@ class ColorgradingGetShadowNode extends LogicNode {
 		super(tree);
 	}
 
-    override function get(from:Int):Dynamic {
-        if(from == 0) {
-            return armory.renderpath.Postprocess.colorgrading_shadow_uniforms[0][1];
```

Comment:
The same goes for this, this is not a typo (check the socket types, the first one is a float output and not a vector!) and also an output got merged like above.

Reply:
Of course this also goes for `colorgrading_global_uniforms`, I forgot that.

Reply:
> The same goes for this, this is not a typo (check the socket types, the first one is a float output and not a vector!) and also an output got merged like above.

I didn't understand well. I need to correct the socket type in the .py of this nodes because they are vectors?

I am also thinking about rename `ColorGrading Get Shadow` Node by `Get CG Shadow`, you think is  a good thing? Or there is some better name design for this nodes?

Reply:
> I didn't understand well. I need to correct the socket type in the .py of this nodes because they are vectors?

No, sorry, I was just referring to the node so that you see what socket output type to expect. You changed it (because you thought this was a typo) to a returned vector instead of a float value.

> I am also thinking about rename ColorGrading Get Shadow Node by Get CG Shadow, you think is a good thing? Or there is some better name design for this nodes?

I wouldn't to this, in this context I wouldn't now what CG means. I will soon investigate if we can set minimum widths for certain nodes, maybe this will resolve some of those naming problems as well (as the name is not cut off anymore)

Reply:
> I will soon investigate if we can set minimum widths for certain nodes

This would be amazing!